### DISCUSSION TITLE
[Question] Scroll doesn't working on Column

### DISCUSSION THREAD

**Discussion by Dzheremi2** [flet-dev/flet#3989] [Q&A] [answered]
### Question

I am adding `scroll=ft.ScrollMode` parameters to `ft.Column`, but none of them are working. I tried container too, but it also didn't work. This scrolling is very important for my app. Global page scroll in this case doesn't work too.

Here you can see that's there more lines, but I can't scroll them
https://github.com/user-attachments/assets/77236d85-9292-4b36-b5a3-38add7b6840b

### Code sample

```python
# This is the function which is creating Column with ScrollMode

def sync_interface(page, lines_cnt, audioplayer):
    lines_column = ft.Column(scroll=ft.ScrollMode.AUTO)
    for i in range(lines_cnt):
        string = split_text_on_lines(page)[i]
        lines_column.controls.append(ft.Row(controls=[syncLine(i, string)]))
    page.remove_at(2)
    page.remove_at(2)
    page.insert(2, lines_column)
    page.controls[0].actions.append(editModeFAB(page, lines_cnt, audioplayer))
    page.add(makeSyncFAB(audioplayer, page))
    page.scroll = None
    page.update()
```

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by HuaxiaGoodBoy** [comment, marked as answer]
You need to specify the height of column

**Reply by Dzheremi2** [reply]
But my column has dynamic height, depended on amount of items inside of it.

**Reply by parmentelat** [reply]
I have been running into this same problem, and stumbled upon
https://stackoverflow.com/questions/77172817/items-not-scrolling-in-flet-gui

with that info I added `expand=True` to the column object, and that did it for me
